src/libsyntax/parse/token.rs

```diff
@@ -147,6 +147,8 @@ pub enum Token {
     Dot,
     DotDot,
     DotDotDot,
+    DotDotEquals,
+    DotEq, // HACK(durka42) never produced by the parser, only used for libproc_macro
```

Comment:
Why is the token called `DotDotEquals`, not `DotDotEq`?

Reply:
Hey, that's a good name! I will probably change it.